ew/static/food.py

```diff
@@ -375,6 +375,28 @@
         str_desc = "Ah, a nice cold brew resembling a mix between battery acid and artificial orange flavoring. Tubular!!",
         time_expir = 43200,
     ),
+  EwFood(
+        id_food = "majormelon",
+        alias = ['rowdydew'],
+        recover_hunger = 10,
+        price = 100,
+        str_name = "Mtn Dew Major Melon",
+        vendors = ['Mtn Dew Fountain', 'vending machine'],
+        str_eat = "You fill your jumbo fountain drink vessel with pink swill and gulp it down.",
+        str_desc = "Ah, a nice cold brew resembling a mix between battery acid and artificial watermelon flavoring. Rowdycore!!",
+        time_expir = 43200,
+	),
+		EwFood(
+        id_food = "purplethunder",
+        alias = ['purpledew'],
+        recover_hunger = 10,
+        price = 100,
+        str_name = "Mtn Dew Purple Thunder",
+        vendors = ['Mtn Dew Fountain', 'vending machine'],
+        str_eat = "You fill your jumbo fountain drink vessel with purple swill and gulp it down.",
+        str_desc = "Ah, a nice cold brew resembling a mix between battery acid and artificial grape flavoring. Killercore!!",
+        time_expir = 43200,
+    ),
```

Comment:
a part of me wants to make these saloon exclusive, since they're not actual mt dew flavors. thoughts?

Reply:
flavours are real and can be bought at Circle K irl. Thank you EbolaSpreadin247 for reminding me to publicly reply